src/responses/yuzu.json

```
@@ -25,6 +25,7 @@
       "shaders": { "reply": "Shader caches store parts of the game when compiling, we don't allow sharing or downloading these, since it is considered piracy."},
       "leaks": { "reply": "The discussion of leaks is not allowed. Talking about the game and its mechanics is fine, talking about the leak and playability is not."},
       "scam": { "reply": "If you believe that a user is attempting to scam you or others, please report them to discord trust and safety and promptly block them. \nTo report the user, please follow this link <https://dis.gd/request>. \nFor more information about how to report them, please look here: <https://support.discord.com/hc/en-us/articles/360000291932-How-to-Properly-Report-Issues-to-Trust-Safety>. \nFinally, if this is the only server you share, please report the user (including screenshots of the conversation) to our moderation team, so that we can address the issue."},
+      "piracy": { "reply": "*Yuzu* is legal, we don't support illegal activities. Dumping your purchased games from your Switch is legal, downloading them is not.\n\nPlease reference the __Quickstart Guide__ in order to dump your games, keys, and system files from your Switch for use with yuzu. <https://yuzu-emu.org/help/quickstart/>\n\nPlease refresh yourself on the <#418589661783392267> you agreed to when you joined the server."},
```

```suggestion
      "piracy": { "reply": "*yuzu* is legal, we don't support illegal activities. Dumping your purchased games and system files from your Switch is legal, downloading them is not.\n\nPlease reference the __Quickstart Guide__ in order to dump your games, keys, and system files from your Switch for use with yuzu. <https://yuzu-emu.org/help/quickstart/>\n\nAlso please refresh yourself on the <#418589661783392267> you agreed to when you joined the server."},
```

We don't have the system files bit over on Citra since you can legally download them through the GUI, provided that you have your AES keys dumped. So it was to reduce potential confusion. Yuzu does not have this though.

---

Updated